cmd/panfs.go

```diff
@@ -162,6 +173,11 @@ func NewPANFSObjectLayer(ctx context.Context, fsPath string) (ObjectLayer, error
 		return nil, err
 	}
 
+	panasasConfigClient := panconfig.NewPanasasConfigClient(
+		env.Get(config.EnvPanasasConfigAgentURL, ""),
```

Comment:
Does it make sense to put this in `cmd/gateway-common.go` and use some global variable instead of get `env` here?